pytorch/buildspec.yml

```diff
@@ -52,6 +52,11 @@ images:
     os_version: &OS_VERSION ubuntu18.04
     tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION ]
     docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+    download_artifacts:
+      awsio:
+        type: s3
+        URI: s3://aws-s3-plugin/binaries/0.0.1/93fdaed/awsio-0.0.1-cp36-cp36m-manylinux1_x86_64.whl
+        VAR_IN_DOCKERFILE: PT_S3_WHL
```

Comment:
Let's merge this PR when this can be removed and we can directly use the public link on the dockerfiles, rather than using the object directly from S3.